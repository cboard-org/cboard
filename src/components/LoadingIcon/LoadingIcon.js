import React from 'react';
import { CircularProgress } from 'material-ui/Progress';
import './LoadingIcon.css';

const LoadingIcon = () => (
  <CircularProgress className="LoadingIcon" size={14} />
);

export default LoadingIcon;
