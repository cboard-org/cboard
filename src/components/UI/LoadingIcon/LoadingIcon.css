.LoadingIcon {
  margin-right: 0.5em;
}
