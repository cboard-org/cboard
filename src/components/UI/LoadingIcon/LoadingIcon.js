import React from 'react';
import CircularProgress from '@material-ui/core/CircularProgress';

import './LoadingIcon.css';

function LoadingIcon() {
  return <CircularProgress className="LoadingIcon" size={14} />;
}

export default LoadingIcon;
