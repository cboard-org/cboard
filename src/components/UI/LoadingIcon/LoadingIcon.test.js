import React from 'react';
import { shallowMatchSnapshot } from '../../../common/test_utils';
import LoadingIcon from './LoadingIcon';

describe('LoadingIcon tests', () => {
  test('default renderer', () => {
    shallowMatchSnapshot(<LoadingIcon />);
  });
});
