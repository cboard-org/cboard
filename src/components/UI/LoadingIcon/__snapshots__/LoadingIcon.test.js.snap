// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIcon tests default renderer 1`] = `
<WithStyles(ForwardRef(CircularProgress))
  className="LoadingIcon"
  size={14}
/>
`;
